src/drivers/evdev/lv_evdev.c

```diff
@@ -229,9 +230,21 @@ static void _evdev_discovery_indev_try_create(const char * file_name)
     lv_indev_t * indev = lv_evdev_create(LV_INDEV_TYPE_NONE, dev_path);
     if(indev == NULL) return;
 
+    lv_evdev_t * dsc = lv_indev_get_driver_data(indev);
+
+    lv_indev_t * ex_indev = NULL;
```

Comment:
Please add a comment here about what this sections does, or wrap it into a function.

Reply:
Thanks!